src/string_id_null_ids.cpp

```diff
@@ -26,6 +26,7 @@ MAKE_NULL_ID( faction, "NULL" )
 MAKE_NULL_ID( ammunition_type, "NULL" )
 MAKE_NULL_ID( vpart_info, "null" )
 MAKE_NULL_ID( emit, "null" )
+MAKE_NULL_ID( vehicle_prototype, "null" )
```

Comment:
Apparently it should be `MAKE_NULL_ID2` as vehicle_prototype is a struct, not a class. It seems not to affect most compilers, but VS would bitch about this when linking.